CONTRIBUTING.md

```diff
@@ -1,8 +1,10 @@
 # Contributing to parse-a-changelog
 
-Thanks for your interest in `parse-a-changelog`. Before contributing, please take a moment to read and sign our <a href="https://github.com/cyberark/parse-a-changelog/blob/master/Contributing_OSS/CyberArk_Open_Source_Contributor_Agreement.pdf" download="conjur_contributor_agreement">Contributor Agreement</a>. This provides patent protection for all `parse-a-changelog` users and allows CyberArk to enforce its license terms. Please email a signed copy to <a href="oss@cyberark.com">oss@cyberark.com</a>.
+Thanks for your interest in `parse-a-changelog`!
 
 For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+In particular, before contributing please review our [contributor licensing guide](https://github.com/cyberark/community/blob/master/CONTRIBUTING.md#when-the-repo-does-not-include-the-cla)
```

Comment:
The piece of the URL after the `#` that specifies a header doesn't seem to do anything. I'm guessing it refers to a header that got removed? We should probably just link to the file without any header specified since they're more prone to changing than the file itself.

Reply:
this PR depends on cyberark/community#74, where that header will be added. it's going to be linked to from a bunch of places, but I commit to changing it everywhere if/when I need to change it again